meshtastic/tests/test_mesh_interface.py

```
@@ -25,6 +30,37 @@
+
+    real_import = builtins.__import__
+
+    def _import_with_print_color_failure(
+        name: str,
+        globals_dict: Any = None,
+        locals_dict: Any = None,
+        from_list: tuple[str, ...] = (),
+        level: int = 0,
+    ) -> Any:
+        if name == "print_color":
+            raise ImportError("simulated missing print_color")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)